cognite/neat/_store/_graph_store.py

```diff
@@ -685,3 +681,8 @@ def _shorten_summary(self, summary: pd.DataFrame) -> pd.DataFrame:
     @property
     def named_graphs(self) -> list[URIRef]:
         return [cast(URIRef, context.identifier) for context in self.dataset.contexts()]
+
+    @property
+    def empty(self) -> bool:
+        """Cheap way to check if the graph store is empty."""
+        return self.queries.has_data()
```

Comment:
Shouldn't this be
```suggestion
    def empty(self) -> bool:
        """Cheap way to check if the graph store is empty."""
        return not self.queries.has_data()
```
That is the store is empyt if it does not have data?